Userland/Utilities/true.cpp

```diff
@@ -4,9 +4,9 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include <sys/cdefs.h>
+#include <LibMain/Main.h>
 
-int main(int, char**)
+ErrorOr<int> serenity_main(Main::Arguments)
```

Comment:
Fwiw true Libmain port has been shut down a few times already, see <https://github.com/SerenityOS/serenity/pull/12380#issuecomment-1033849124>. Perhaps the list should be updated to note that? Or can discuss it again 🙂

Reply:
I think uniformity is a lot more important than a 0.7ms increase in runtime... I'm happy to ~~fight~~ discuss with folks.

Reply:
700 us increase in time for everything is kinda nasty. Maybe we should track down exactly why that is happening and fix it. Is it due to dynamically linking LibMain? That is where I would explore first.

Reply:
We can use the `UserspaceEmulator` profiling support to diff the runtime if the two versions instruction by instruction as well.

Reply:
I agree with Brian here, uniformity is way higher on the list than squeezing `/bin/true` for startup performance.